src/Raven.Server/RavenServer.cs

```diff
@@ -1071,6 +1071,12 @@ private async Task DoActualCertificateRefresh(CertificateUtils.CertificateHolder
                 if (newCertBytes == null)
                     return;
 
+                if (Logger.IsOperationsEnabled)
+                {
+                    var source = Configuration.Core.SetupMode == SetupMode.LetsEncrypt ? "Let's Encrypt" : $"executable ({Configuration.Security.CertificateRenewExec} {Configuration.Security.CertificateRenewExecArguments})";
+                    Logger.Operations($"Got new certificate from {source}. Starting certificate replication.");
```

Comment:
I think to avoid back and forth juggling for that, we could introduce a helper e.g. `CertificateHelper.GetBasicCertificateInfo(certificate)` which would yield a string like '{thumbprint} } | {subject}'

Reply:
In this place, I only log the source.
Before we sometimes mistakenly log the source as LetsEncrypt.
The check relay on the `load` scipt if it was defined or not.
Inside the function `StartCertificateReplicationAsync`  after creating the `X509Certificate2` I log the certificate information.
Currently, I log only the thumbprint and I will add the "subject" as well.